contrib/pai_vscode/package.json

```diff
@@ -417,13 +430,15 @@
         "@types/uuid": "^3.4.4",
         "nock": "^10.0.1",
         "tslint": "^5.8.0",
-        "tslint-language-service": "^0.9.9",
         "tslint-microsoft-contrib": "^5.2.1",
-        "typescript": "^3.1.1",
+        "typescript": "^3.5.1",
+        "typescript-tslint-plugin": "^0.4.0",
         "vsce": "^1.53.0",
         "vscode": "^1.1.21"
     },
     "dependencies": {
+        "@types/js-yaml": "^3.12.1",
```

Comment:
You need to install both the type definition file `@types/js-yaml` and the javascript lib `js-yaml` itself.

Reply:
Done.